docs/rules-and-regulations.md

```diff
@@ -17,204 +16,101 @@ The Organization shall not arbitrarily discriminate on the basis of race, nation
 
 The Organization shall have three classes of members: 
 1. Partners (Organizational Owners)
-2. Contributor 
-3. Community members
+2. Contributor
+3. Community
+4. Network
  
 ### Section 2.02. Membership Qualifications.  
 
-Any person may become and remain a member of this Organization by:
-1. Being sponsored by an existing member; and
-2. Meeting the qualifications set forth for each Membership Classification; and  
-3. Making full payment of any non-refundable membership fee as set forth in Section 2.06 of these Rules and Regulations. 
+Any person may become and remain a member of this Organization.
```

Comment:
I think this needs to be separated into parts. TBD.  Good until the next iteration.

To consider: 
1.  Any person may become a member of this Organization (Collective)
2.  Defining Membership in good standing (active)(inactive), criteria to be in bad standing needs to be defined in a code of conduct section.
3.  What is the code of conduct, and what are the consequences.